### ISSUE TITLE
[JOSS] Missing or renamed file?

### ISSUE THREAD

**Issue by RMeli** [Degiacomi-Lab/molearn_notebook#2]
When running the `minimal_example` notebook, I get the following error:
```
Exception: ERROR: file data/MurD_closed_open_selection.pdb not found!
```

I extracted the files from the `data/*.tar.gz` archives, but I only have the following PDB files:
```
MurD_closed_apo_selection.pdb  MurD_closed_selection.pdb  MurD_open_selection.pdb
```
They correspond to the names of the archives.

However, running the notebook requires the following files:
```python
networkfile = f'data{os.sep}checkpoint_no_optimizer_state_dict_epoch167_loss0.003259085263643.ckpt'
training_set_files = [f'data{os.sep}MurD_open_selection.pdb', f'data{os.sep}MurD_closed_open_selection.pdb']
test_set_file = f'data{os.sep}MurD_closed_apo_selection.pdb'
```

Is `data{os.sep}MurD_closed_open_selection.pdb` missing or it has been simply renamed to `MurD_closed_selection.pdb`?

---

https://github.com/openjournals/joss-reviews/issues/5523

**Comment by degiacom** [comment]
Apologies, the input data had been renamed.  The training set is built with two files,  `MurD_closed_selection.pdb`, and `MurD_open_selection.pdb`. The test set is `MurD_closed_apo_selection.pdb`, that is a trajectory of MurD switching from the closed to the open state upon removal of its ligand. `minimal_example.ipynb` has been updated accordingly.

**Comment by RMeli** [comment]
Thanks for the clarification!
